BitwardenShared/UI/Tools/Send/SendItem/AddEditSendItem/AddEditSendItemView.swift

```diff
@@ -392,14 +395,20 @@ struct AddEditSendItemView: View { // swiftlint:disable:this type_body_length
         )
         .textFieldConfiguration(.password)
 
-        BitwardenMultilineTextField(
+        BitwardenField(
             title: Localizations.notes,
-            text: store.binding(
-                get: \.notes,
-                send: AddEditSendItemAction.notesChanged
-            ),
             footer: Localizations.notesInfo
-        )
+        ) {
+            BitwardenUITextView(
+                text: store.binding(
+                    get: \.notes,
+                    send: AddEditSendItemAction.notesChanged
+                ),
+                calculatedHeight: $notesDynamicHeight
+            )
+            .frame(minHeight: notesDynamicHeight)
+            .accessibilityLabel(Localizations.notes)
```

Comment:
❓ There's two multiline text fields in the add/edit sends view. This one, but then also the text of the send. Should we also apply these changes to the "Text" field?

https://github.com/bitwarden/ios/blob/8cfacf4bf19f981fd2d1b7f71bf88b61ea9dcb22/BitwardenShared/UI/Tools/Send/SendItem/AddEditSendItem/AddEditSendItemView.swift#L471-L479

That seems to be the last use of `BitwardenMultilineTextField`, should we remove that?

Reply:
good catch.   yeah, i don't see a need for `BitwardenMultilineTextField` if the UITextField implementation exists and is more usable from a user stand point.